internal/scaffold/constants.go

```diff
@@ -23,15 +23,17 @@ const (
 	filePathSep = string(filepath.Separator)
 
 	// dirs
-	CmdDir         = "cmd"
-	ManagerDir     = CmdDir + filePathSep + "manager"
-	PkgDir         = "pkg"
-	ApisDir        = PkgDir + filePathSep + "apis"
-	ControllerDir  = PkgDir + filePathSep + "controller"
-	BuildDir       = "build"
-	BuildTestDir   = BuildDir + filePathSep + "test-framework"
-	BuildBinDir    = BuildDir + filePathSep + "_output" + filePathSep + "bin"
-	BuildScriptDir = BuildDir + filePathSep + "bin"
-	DeployDir      = "deploy"
-	CRDsDir        = DeployDir + filePathSep + "crds"
+	CmdDir        = "cmd"
+	ManagerDir    = CmdDir + filePathSep + "manager"
+	PkgDir        = "pkg"
+	ApisDir       = PkgDir + filePathSep + "apis"
+	ControllerDir = PkgDir + filePathSep + "controller"
+	BuildDir      = "build"
+	BuildBinDir   = BuildDir + filePathSep + "_output" + filePathSep + "bin"
+	DeployDir     = "deploy"
+	CRDsDir       = DeployDir + filePathSep + "crds"
+
+	// files
+	DockerfileFile   = "Dockerfile"
+	OperatorYamlFile = "operator.yaml"
```

Comment:
we will be able to remove the vars related to the legacy layout. However, it is not in the scope of this pr.